IdentityCore/src/webview/embeddedWebview/ui/ios/MSIDWebviewUIController.m

```diff
@@ -53,11 +53,6 @@ + (void)initialize
 + (WKWebViewConfiguration *)defaultWKWebviewConfiguration
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->